test/jasmine/assets/custom_matchers.js

```diff
@@ -127,7 +127,7 @@ var matchers = {
                     'to be close to',
                     arrayToStr(expected.map(arrayToStr)),
                     msgExtra
-                ].join(' ');
+                ].join('\n');
```

Comment:
This way the two arrays always start at the same column, so it's easy to match points and see what failed.